code/API_definitions/device-roaming-status-subscriptions.yaml

```diff
@@ -498,36 +704,9 @@ components:
         - org.camaraproject.roaming-status-subscriptions.v0.roaming-on
```

Comment:
I guess we need here device- too, and then in the scopes too.

Reply:
Aggree 🚀